modules/db_instance/main.tf

```diff
@@ -78,6 +78,19 @@ resource "aws_db_instance" "this" {
   deletion_protection      = var.deletion_protection
   delete_automated_backups = var.delete_automated_backups
 
+
+  dynamic "restore_to_point_in_time" {
+
+    for_each = var.restore_to_point_in_time != null ? [var.restore_to_point_in_time] : []
+    content {
+      restore_time                  = restore_to_point_in_time.value.restore_time
+      source_db_instance_identifier = lookup(restore_to_point_in_time.value, "source_db_instance_identifier", null)
+      source_dbi_resource_id        = lookup(restore_to_point_in_time.value, "source_dbi_resource_id", null)
+      use_latest_restorable_time    = lookup(restore_to_point_in_time.value, "use_latest_restorable+time", null)
```

Comment:
There is a typo here - we have `use_latest_restorable+time` but should be `use_latest_restorable_time`

Reply:
fixed